ensembl_dbcopy/forms.py

```diff
@@ -87,41 +104,95 @@ def clean_tgt_host(self):
         return data
 
     def clean_src_incl_db(self):
-        data = self.cleaned_data['src_incl_db'].rstrip(',')
-        return data
+        return self._cleaned_text_field('src_incl_db')
 
     def clean_src_skip_db(self):
-        data = self.cleaned_data['src_skip_db'].rstrip(',')
-        return data
+        src_skip_db = self._cleaned_text_field('src_skip_db')
+        tgt_db_name = self._cleaned_text_field('tgt_db_name')
+        src_skip_db_names = _text_field_as_set(src_skip_db)
+        tgt_db_names = _text_field_as_set(tgt_db_name)
+        if src_skip_db_names and tgt_db_names:
```

Comment:
Same here: could use a dedicated validator on field.

Reply:
Okay. To be honest, I haven't completely understood the practical difference between _validation_ and _cleaning_ when it comes to django forms. I'll look into it.

Reply:
I must admit the distinction is not crystal clear :-) How I see it: clean is supposed to be very "field logic specific" and validators are more "field content specific", but basically the same "process inside". 
Example: Validator on "regExp" or "even/odd" content, Src_host: trim the field first, check that it's a list of text separated by `,` chars.